crates/pkgcruft/Cargo.toml

```diff
@@ -62,17 +60,19 @@ tracing-subscriber = "0.3.19"
 tree-sitter = "0.25.4"
 url = "2.5.4"
 walkdir = "2.5.0"
-
-# exported test support deps
 assert_cmd = { version = "2.0.17", optional = true }
 glob = { version = "0.3.2", optional = true }
 
+
```

Comment:
Not sure why the extra space is needed here.

Reply:
not needed